CONTRIBUTING.md

```diff
@@ -12,6 +12,22 @@ We adhere to the following guidelines:
 - Code Formatting: [fmt-rfcs Rust Style Guide]
 - API building: [Rust API Guidelines]
 
+# Code Formatting
+
+The code formatting should match exactly with the behavior of `rustfmt` on nightly, though, so
```

Comment:
Why not just outright require `rustfmt` to be ran on the code?
Also, I don't know what type of CI you have on the repo, but it might be integrated into it.

Reply:
Requiring it be run sounds like a good idea.

The main reason I haven't added a CI check is just because of how fast `rustfmt` style can change. I haven't wanted to deny a PR just because some unrelated portion of code would be formatted differently in the newest version. If we're active enough, though, that could be avoided by regular "reformat" prs and requiring people to rebase.

Reply:
I'm down for it. Let's ride the format train!

Reply:
:+1: 

If we want to do the travis part I'll wait until we merge #40 to do that (or add that to #40).